src/app/components/case/case-retention-date/case-retention-date.component.html

```diff
@@ -80,14 +80,31 @@ <h2 class="govuk-heading-m">Retention audit history</h2>
     </ng-container>
   </ng-container>
   <!-- Summary / Confirmation Screen -->
-  <div *ngIf="state === 'Change'" class="order-confirmation">
+  <div *ngIf="state === 'Change' || state === 'Confirm'" [hidden]="state === 'Confirm'" class="order-confirmation">
     <app-case-retention-change
       [state]="state"
       [currentRetentionDate]="vm.caseDetails.case_retain_until_date_time"
       [originalRetentionDate]="getOriginalRetentionDateString(vm.retentionHistory)"
       (stateChange)="onStateChanged($event)"
+      (retentionDateChange)="onRetentionDateChanged($event)"
+      (retentionReasonChange)="onRetentionReasonChanged($event)"
+      (retentionPermanentChange)="onRetentionPermanentChanged($event)"
     ></app-case-retention-change>
   </div>
+  <div *ngIf="state === 'Change' || state === 'Confirm'" [hidden]="state === 'Change'" class="order-confirmation">
+    <app-case-retention-confirm
+      [state]="state"
+      [newRetentionDate]="newRetentionDate"
```

Comment:
Few dupes here similar to 1437

Reply:
Just merged master into this one, should be fixed now